clients/src/main/java/org/apache/kafka/common/security/JaasUtils.java

```diff
@@ -88,25 +87,17 @@ public static boolean isZkSecurityEnabled() {
         boolean zkSaslEnabled = Boolean.parseBoolean(System.getProperty(ZK_SASL_CLIENT, "true"));
         String zkLoginContextName = System.getProperty(ZK_LOGIN_CONTEXT_NAME_KEY, "Client");
 
-        String loginConfigFile = System.getProperty(JAVA_LOGIN_CONFIG_PARAM);
-        if (loginConfigFile != null && loginConfigFile.length() > 0) {
```

Comment:
@rajinisivaram It sounds ok to remove this verification, if the configuration isn't right, then isSecurityEnabled below will be false, is that right?